http/src/main/scala/org/broadinstitute/dsde/workbench/leonardo/util/VPCHelper.scala

```diff
@@ -12,33 +14,34 @@ import org.broadinstitute.dsde.workbench.model.google.GoogleProject
 
 import scala.collection.JavaConverters._
 
-class VPCHelper(config: VPCHelperConfig,
-                googleProjectDAO: GoogleProjectDAO,
-                googleComputeService: GoogleComputeService[IO])(implicit contextShift: ContextShift[IO]
-                                                                //  log: Logger[IO]
-) {
+class VPCHelper[F[_]: Async: ContextShift: Logger](config: VPCHelperConfig,
+                                                   googleProjectDAO: GoogleProjectDAO,
+                                                   googleComputeService: GoogleComputeService[F],
+                                                   blocker: Blocker) {
 
-  private def getVPCSettingsFromProjectLabel(googleProject: GoogleProject): IO[Option[VPCConfig]] =
-    IO.fromFuture(IO(googleProjectDAO.getLabels(googleProject.value))).map { labelMap =>
+  implicit val contextShiftIO = IO.contextShift(blocker.blockingContext)
```

Comment:
This probably isn't correct.. basically for `IO.fromFuture(…)` I need a `ContextShift[IO]`. But I only have a `ContextShift[F]`. I couldn't figure out how to create a `ContextShift[IO]` from a `ContextShift[F]` so I defined one in this way using the blocking context.

However now I think it might be better for this class to take a `ContextShift[IO]` in its constructor and have `Boot`/`IOApp` provide it.